src/styles.css

```
@@ -0,0 +1,244 @@
+
+html, body {
+    margin: 0;
+    padding: 0;
+    height: 100%;
+    font-family: 'Roboto', sans-serif;
+}
+/* contenedor principal */
+.container {
+    width: 100%;
+    height: 100%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+/* header */
+header {
+    padding: 0.2rem;
+    background: rgb(47, 141, 218);
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+}
+.codeMakers {width: 12rem; height: 4rem;}
+.viewsStyle {
+    width: 90%;
+}
+.views {
+    width: 90%;
+    height: 100%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+/* iniciar sesion con fb y google */
+.conectWithSocialMedia {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+.buttons{
+    border: 0;
+    height: 4rem;
+    margin: 0;
+  }
+/* formulario */
+.registerForm {
+    width: 100%;
+    height: 50%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: space-around;
+}
+.recoveryForm {
+    width: 100%;;
+    height: 30%;
+    margin-bottom: 2rem;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: space-around;
+}
+input {
+    border: 0;
+    width: 100%;
+    height: 2rem;
+    background: linear-gradient(0deg, rgba(154,156,156,1) 0%, rgba(245,245,242,1) 100%);
+}
+button {
+    width: 100%;
+    height: 2.5rem;
+    border: 0;
+}
+.registerButton {
+    border: 0;
+    background: rgb(47, 141, 218);
+    border-radius: 18px 18px 18px 18px;
+    border: 6px outset rgb(47, 141, 218);
+    font-weight: bold;
+}
+
+
+/* vista de home */
+.Home {
+    width: 112%;
+    height: auto;
+}
+.homeOptions {
+    width: 100%;
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+}
+    .btnHeader {width: 2rem; height: 2rem;}
+.sesionPrincipal, .sesionPrincipal1 {
+    padding: 0;
+    margin: 1rem 0;
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+.newPost {
+    width: 90%; 
+    align-content: center; 
+    display: flex;
+    flex-direction: column;
+}
+.welcome {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin: 2rem 0;
+}
+.postForm {
+    width: 100%;
+    resize: none;
+}
+
+hr {width: 90%; border-color: rgb(47, 141, 218);}
+.postContainer {
+    margin: 0.5rem 0.3rem;
+    width: 90%; 
+    padding: 0.5rem;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    border-radius: 26px 26px 26px 26px;
+    border: 0.25rem dashed #bdb5bd;
+    background: rgb(243, 243, 243);
+}
+    .post {
+        display: flex;
+        width: 100%;
+        justify-content: space-around;
+    }
+        .postAuthor {
+            width: 4rem; 
+            height: 4rem;
+        }
+        .postContent {
+            width: 85%; 
+            height: auto;
```

al definir width y asignarle un valor, puedes omitir height.
```suggestion
            width: 85%; 
         
```